implementations.md

```diff
@@ -11,6 +11,7 @@ If you know of any associated projects that are not listed here, please file a p
 
 * [hyperhq/runv][runv] - Hypervisor-based runtime for OCI
 * [01org/cc-oci-runtime][cc-oci] - Hypervisor-based OCI runtime for Intel® Architecture
+* [clearcontainers/runtime][cc-runtime] - Next-generation hypervisor-based OCI runtime for Intel® Architecture
```

Comment:
Just wondering, what's the difference between cc-runtime and cc-oci-runtime?

Reply:
Hi @hqhq - I've now removed [`cc-oci-runtime`](https://github.com/01org/cc-oci-runtime) as that project is now in maintenance mode.

[`cc-runtime`](https://github.com/clearcontainers/runtime) is a complete rewrite in golang (under an Apache license rather than GPL). See the main page for further details.